examples/coherence/src/main/java/io/helidon/examples/coherence/CreditscoreService.java

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright (c) 2025 Oracle and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.helidon.examples.coherence;
+
+import io.helidon.http.Status;
+import io.helidon.webserver.http.HttpRules;
+import io.helidon.webserver.http.HttpService;
+import io.helidon.webserver.http.ServerRequest;
+import io.helidon.webserver.http.ServerResponse;
+
+import com.tangosol.net.CacheFactory;
+import com.tangosol.net.NamedCache;
+
+import static java.lang.System.Logger.Level.INFO;
+
+/**
+ * Credit score service.
+ */
+public class CreditscoreService implements HttpService {
+
+    private static final System.Logger LOGGER = System.getLogger(CreditscoreService.class.getName());
+    private static final int SCORE_MAX = 800;
+    private static final int SCORE_MIN = 550;
+
+    private final NamedCache<String, Integer> creditScoreCache = CacheFactory.getCache("creditScoreCache");
+
+    @Override
+    public void routing(HttpRules rules) {
+        rules.post(this::postCreditscore);
+    }
+
+    private void postCreditscore(ServerRequest req, ServerResponse res) {
```

Comment:
In my opinion, the camel case name would look better here.

Reply:
Done

Reply:
Done